src/backend/egl/display.rs

```diff
@@ -799,7 +799,17 @@ fn get_dmabuf_formats(
 
     for fourcc in formats {
         let mut num = 0i32;
-        wrap_egl_call(|| unsafe {
+        // Some drivers return EGL_BAD_PARAMETER here for formats
+        // they themselves returned for the query above.. *sigh*
+        //
+        // - NVIDIA proprietary since 520
+        //
+        // So lets ignore any errors of this call on purpose(!),
+        // which will let `num` stay at `0` and handle the format
+        // as unsupported by explicit modifiers.
+        // Which is probably what the error is suppose to indicate
+        // although the spec doesn't seem to demand it...
+        unsafe {
             ffi::egl::QueryDmaBufModifiersEXT(
```

Comment:
Is the only error that can be returned `EGL_BAD_PARAMETER` or are we actually interessted in other errors?
Should we explicitely handle that one error and maybe also explicitely set `num = 0` in that case or is num guaranteed to be untouched if an error occured?
Or maybe we want to at least log that error?

Reply:
You are right, some safety percussions cannot hurt.

Does this look better?

Reply:
👍

Reply:
The EGL specification says that the function will return EGL_FALSE on failure